etc/systemd/system/chia-monitor.service

```diff
@@ -0,0 +1,134 @@
+#-- Alerts   : https://github.com/caronc/apprise/wiki
+#-- Requires : chia python3 pipenv git rsyslog
+#------------:
+#-- Date     : 2021-08-28
+#-- Compat   : Tested w/ chia-monitor commit 9472b6c5adbf9c47f9ec9e4ab7f9ac73f79c64bf / 2021-08-26
+#-- Version  : 1.0
+#------------:
+#-- Dev Todo : 1) Consider adding systemd notifier top-level drop in command for service outage webhook exec
+#--          : 2) Add a [Unit] call for "After=chia-proc.service" if/when Chia is run as a service
+#----------------------------------------------------------------------------------------------------------#
+# Base: operating directory and git clone will be as follows:
+#       - /opt/chia-monitor
+#
+# Note: Chia process owner is assumed to be named 'chia-proc', replace as needed for your deployment
+#       - proc user will be the one running the main GUI or CLI farming processes
+#       - proc user will be the owner of the main chia config file
+#       - exporters should run as a separate user from the monitored process in general, however...
+#       - if you want chia-monitor.service running as the chia processes owner, modify these commands
+#       - to allow chia-monitor access to the chia config we will create a shared group for the users
+#
+# Create shared group for 'chia-monitor' and 'chia-proc' related user accounts
+#   sudo groupadd --gid 8001 chia
+#
+# User creation for chia-monitor service
+#   sudo groupadd --gid 8000 chia-monitor
+#   sudo useradd -d /opt/chia-monitor -g chia-monitor -M --shell /sbin/nologin --uid 8000 chia-monitor
+#
+# Group access mods for chia-monitor and chia-proc users
+#   sudo usermod --groups chia --append chia-monitor
+#   sudo usermod --groups chia --append chia-proc
+#   sudo chmod 770 /home/chia-proc
+#   sudo chown chia-proc:chia /home/chia-proc
+#   sudo chown -R chia-proc:chia /home/chia-proc/.chia
+#   sudo chmod 640 /home/chia-proc/.chia/mainnet/config/config.yaml
+#
+# Create operating environment for chia-monitor
+#   sudo cd /opt && git clone https://github.com/philippnormann/chia-monitor.git && cd /opt/chia-monitor
+#   sudo chown -R chia-monitor:chia /opt/chia-monitor
+#   sudo -u chia-monitor /usr/local/bin/pipenv install
+#   sudo -u chia-monitor /usr/local/bin/pipenv run alembic upgrade head
+#   sudo -u chia-monitor cp config-example.json config.json
+#   <edit config.json>
+#
+# Run chia-monitor command to test
+#   <ensure primary .chia directory and config is read-able by chia-monitor user according to previous commands>
+#   sudo -u chia-monitor /usr/local/bin/pipenv run python -m monitor
+#   <if chia-monitor is working correctly, ctrl-c to kill the test>
+#
+# Setup rsyslog config for chia-monitor to avoid system log flooding
+#   sudo touch /var/log/chia-monitor.log
+#   sudo cp etc/rsyslog.d/chia-monitor.conf /etc/rsyslog.d/
+#   sudo systemctl restart rsyslog
+#
+# Execute systemd commands for chia-monitor.service
+#   sudo cp chia-monitor.service /etc/systemd/system/
```

Comment:
`chia-monitor.service` should be `etc/systemd/system/chia-monitor.service` since we are at the project root